cloudflare.go

```diff
@@ -116,12 +116,17 @@ func (api *API) makeRequest(method, uri string, params interface{}) ([]byte, err
 	return resBody, nil
 }
 
+type ResponseInfo struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+}
+
 // Response is a template.  There will also be a result struct.  There will be a
 // unique response type for each response, which will include this type.
 type Response struct {
-	Success  bool     `json:"success"`
-	Errors   []string `json:"errors"`
-	Messages []string `json:"messages"`
+	Success  bool           `json:"success"`
+	Errors   []ResponseInfo `json:"errors"`
+	Messages []ResponseInfo `json:"messages"`
```

Comment:
Is `messages` always a `[]string` or does it mirror `errors`? https://api.cloudflare.com/#responses